pkg/kubelet/server/portforward/.import-restrictions

```diff
@@ -0,0 +1,11 @@
+{
+	"Rules": [
+	  {
+		"SelectorRegexp": "k8s[.]io/kubernetes",
+		"AllowedPrefixes": [
+		],
+		"ForbiddenPrefixes": [
+		]
```

Comment:
what do restrictions with empty allowed and forbidden lists do?

Reply:
empty AllowedPrefixes implies there is nothing that is allowed. technically i can drop ForbiddenPrefixes from here as it has no effect. will do that shortly